lib/reporters/html/index.js

```diff
@@ -38,9 +38,9 @@ function makeDirFor(destPath) {
     return fs.mkdirsAsync(path.dirname(destPath));
 }
 
-function prepareViewData(runner) {
+function prepareViewData(runner, reporterOptions) {
     return new Promise((resolve) => {
-        var model = new ViewModel(runner.config);
+        var model = new ViewModel(runner.config, reporterOptions);
```

Comment:
You shouldn't pass this options to viewModel only for skip addSuccess result. You can do it here - on handler of `TEST_RESULT` event.